src/main.js

```diff
@@ -0,0 +1,40 @@
+const { scan } = require("@onebeyond/license-checker/src/runner");
+const core = require("@actions/core");
+
+async function run() {
+  try {
+    const timestamp = new Date().toISOString();
+
+    // TODO: Input validation
+    const options = {
+      // `failOn` input: fail (exit with code 1) if any package license does not satisfies any license in the provided list
+      failOn: core.getInput("failOn", { required: true }).split(","),
```

Comment:
I think we could add some simple input validation for this field to check it is a "comma separated" list of strings, instead of expecting it to fail. That way we can provide a much more detailed error message. What do you think.

Reply:
I agree. It would be great to let the user know why it is failing.

Reply:
And to avoid a long file, it would also be good to move that validation to a separate file?